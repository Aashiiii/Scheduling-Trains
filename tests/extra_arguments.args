Clyde Turella 22:10:10 Hello
