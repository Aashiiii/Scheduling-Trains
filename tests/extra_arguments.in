Clyde::Turella::22:10:10
Werrington::Vineyard::11:12:11
Waverton::Vineyard::00:00:01
Campsie::Vineyard::23:45:43
