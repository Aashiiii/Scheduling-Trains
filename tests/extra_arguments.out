Too many arguments. Please provide <source> <destination> <time> as command line arguments
